website/docs/api/spancategorizer.md

```diff
@@ -60,17 +87,26 @@ architectures and their arguments and hyperparameters.
 | `threshold`    | Minimum probability to consider a prediction positive. Spans with a positive prediction will be saved on the Doc. Defaults to `0.5`. ~~float~~                                                                                                                                                          |
 | `max_positive` | Maximum number of labels to consider positive per span. Defaults to `None`, indicating no limit. ~~Optional[int]~~                                                                                                                                                                                      |
 | `scorer`       | The scoring method. Defaults to [`Scorer.score_spans`](/api/scorer#score_spans) for `Doc.spans[spans_key]` with overlapping spans allowed. ~~Optional[Callable]~~                                                                                                                                       |
+| `negative_weight`    | Multiplier for the loss terms. It can be used to down weigh the negative samples if there are too many. It is only available when using the `spancat_exclusive` component. ~~float~~                                                  |
+| `allow_overlap`        | If `True`, the data is assumed to contain overlapping spans. It is only available when using the `spancat_exclusive` component. ~~bool~~ |
+
+
 
 ```python
 %%GITHUB_SPACY/spacy/pipeline/spancat.py
 ```
 
+```python
+%%GITHUB_SPACY/spacy/pipeline/spancat_exclusive.py
+```
+
 ## SpanCategorizer.\_\_init\_\_ {#init tag="method"}
 
 > #### Example
 >
 > ```python
 > # Construction via add_pipe with default model
+> # Use 'spancat_exclusive' for exclusive clases
```

Comment:
```suggestion
> # Use 'spancat_exclusive' for exclusive classes
```

(And I'm not sure users will understand what "for exclusive classes" means here.)